core/src/avm2/object/bitmapdata_object.rs

```diff
@@ -62,6 +61,11 @@ pub struct BitmapDataObjectData<'gc> {
     bitmap_data: Lock<Option<BitmapDataWrapper<'gc>>>,
 }
 
+const _: () = assert!(std::mem::offset_of!(BitmapDataObjectData, base) == 0);
```

Comment:
can these be moved to inside the method?
(Maybe even generated by a macro, to hide the repeated `unsafe`? ;D )

Reply:
I think it makes more sense to have them here, as they're referencing the struct immediately above, and the TObject impl isn't right after the `struct AbcObjectData` definition for several object variants.

Reply:
My motivation for this was that them being directly above an `unsafe` block explains its safety better, but I'm not hell bent on this :)